libraries/provider_component_runit_supervisor_upstart.rb

```diff
@@ -0,0 +1,71 @@
+require "chef/provider/lwrp_base"
+
+class Chef
+  class Provider
+    class ComponentRunitSupervisor
+      class Upstart < Chef::Provider::LWRPBase
+        provides :component_runit_supervisor, :platform_family => "debian"
+        provides :component_runit_supervisor, :platform_family => "rhel" do |node|
+          node['platform_version'] =~ /^6/
```

Comment:
Got it. That's what I would have done in the first place but it was copying from [what was there before](#diff-87632c3c644e61ff3846321eee72300bL29).